app/Console/Kernel.php

```diff
@@ -13,26 +13,7 @@ class Kernel extends ConsoleKernel
      * @var array
      */
     protected $commands = [
-        Commands\PaveIt::class,
```

Comment:
I like this change, but I think I'd like to see it in a separate cleanup commit, not here.

Reply:
Put back commands, will do separate PR for autoloading.